karrot/subscriptions/receivers.py

```diff
@@ -211,8 +211,8 @@ def send_group_detail(group, user=None):
 
 
 def send_group_preview(group):
-    preview_payload = GroupPreviewSerializer(group).data
     for subscription in ChannelSubscription.objects.recent():
+        preview_payload = GroupPreviewSerializer(group, context={'request': MockRequest(user=subscription.user)}).data
```

Comment:
GroupPreviewSerializer needs request user to work properly, this was broken before but uncritical (delivered invalid `isMember` fields via websocket).

Reply:
Ah, good catch! :star: